web/client/stores/StandardStore.js

```diff
@@ -68,6 +69,7 @@ module.exports = (initialState = {defaultState: {}, mobile: {}}, appReducers = {
                 loadingError: mapState.loadingError,
                 mapId: mapState.loadingError.mapId
             } || null,
+            mapConfigRawData: mapState && mapState.mapConfigRawData || null,
```

Comment:
It think this have to be removed

Reply:
@offtherailz without this i can't store in this field in redux anything, even when i made proper action and dispatch it